src/Extensions/Hosting/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Omex.Extensions.Logging;
+using Microsoft.Omex.Extensions.TimedScopes;
+
+namespace Microsoft.Omex.Extensions.Hosting
+{
+	/// <summary>
+	/// Extension to add Omex dependencies to HostBuilder
+	/// </summary>
+	public static class HostBuilderExtensions
+	{
+		/// <summary>
+		/// Add Omex Logging and TimedScopes dependencies
+		/// </summary>
+		public static IHostBuilder AddOmexServices(this IHostBuilder builder) =>
```

Comment:
Validate the parameter isn't `null` for this method and the one below.

Reply:
Never mind. I forgot you modified the project options to disallow `null`s.

Reply:
Even if we manage to squeeze null here extension methods will throw an identical exception